static/app/views/dashboardsV2/datasetConfig/releases.tsx

```diff
@@ -170,3 +193,133 @@ export function transformSessionsResponseToSeries(
 
   return results;
 }
+
+function fieldsToDerivedMetrics(field: string): string {
+  return FIELD_TO_METRICS_EXPRESSION[field] ?? field;
+}
+
+function getRequest(
```

Comment:
nit: can we name this to something less generic :P `getSessionsRequest` or something?